src/main/kotlin/no/nav/syfo/identhendelse/database/IdenthendelseQuery.kt

```diff
@@ -0,0 +1,132 @@
+package no.nav.syfo.identhendelse.database
+
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.application.database.toList
+import no.nav.syfo.dialogmotekandidat.database.PDialogmotekandidatEndring
+import no.nav.syfo.dialogmotekandidat.database.PDialogmotekandidatStoppunkt
+import no.nav.syfo.dialogmotestatusendring.database.PDialogmoteStatus
+import no.nav.syfo.domain.PersonIdentNumber
+import no.nav.syfo.unntak.database.domain.PUnntak
+import java.sql.ResultSet
+import java.time.OffsetDateTime
+import java.util.*
+
+const val queryUpdateKandidatStoppunkt =
+    """
+        UPDATE DIALOGMOTEKANDIDAT_STOPPUNKT
+        SET personident = ?
+        WHERE personident = ?
+    """
+
+fun DatabaseInterface.updateKandidatStoppunkt(nyPersonident: PersonIdentNumber, stoppunktWithOldIdent: List<PDialogmotekandidatStoppunkt>): Int {
+    var updatedRows = 0
+    this.connection.use { connection ->
+        connection.prepareStatement(queryUpdateKandidatStoppunkt).use {
+            stoppunktWithOldIdent.forEach { stoppunkt ->
+                it.setString(1, nyPersonident.value)
+                it.setString(2, stoppunkt.personIdent.value)
+                it.executeUpdate()
+                updatedRows++
+            }
+        }
+        connection.commit()
+    }
+    return updatedRows
+}
+
+const val queryUpdateKandidatEndring =
+    """
+        UPDATE DIALOGMOTEKANDIDAT_ENDRING
+        SET personident = ?
+        WHERE personident = ?
+    """
+
+fun DatabaseInterface.updateKandidatEndring(nyPersonident: PersonIdentNumber, endringWithOldIdent: List<PDialogmotekandidatEndring>): Int {
+    var updatedRows = 0
+    this.connection.use { connection ->
+        connection.prepareStatement(queryUpdateKandidatEndring).use {
+            endringWithOldIdent.forEach { endring ->
+                it.setString(1, nyPersonident.value)
+                it.setString(2, endring.personIdent.value)
+                it.executeUpdate()
+                updatedRows++
+            }
+        }
+        connection.commit()
+    }
+    return updatedRows
+}
+
+const val queryUpdateUnntak =
+    """
+        UPDATE UNNTAK
+        SET personident = ?
+        WHERE personident = ?
+    """
+
+fun DatabaseInterface.updateUnntak(nyPersonident: PersonIdentNumber, unntakWithOldIdent: List<PUnntak>): Int {
+    var updatedRows = 0
+    this.connection.use { connection ->
+        connection.prepareStatement(queryUpdateUnntak).use {
+            unntakWithOldIdent.forEach { unntak ->
+                it.setString(1, nyPersonident.value)
+                it.setString(2, unntak.personIdent)
+                it.executeUpdate()
+                updatedRows++
+            }
+        }
+        connection.commit()
+    }
+    return updatedRows
+}
+
+const val queryGetDialogmoteStatus =
```

Comment:
Kan vurdere å endre dette til bare å hente ut antallet, dvs select count(id)

Siden oppdateringen av ident skjer for alle radene med den gamle ident'en på en gang, så trenger man egentlig ikke selecte ut alle de aktuelle radene og instansiere dem opp som pobjekter.

Reply:
Aha, ja det var lurt!

Reply:
Hmm, men det vi mister da er hvilke av de gamle identene vi faktisk trenger å oppdatere 🤔 
Hvis en person har 5 gamle identer, og vi har 2 av de i databasen, så ønsker vi jo egt bare å kjøre en update mot de 2. Vi får vel ingen feil for å gjøre update mot 5, da får vi vel bare "Zero rows affected" eller noe sånt. Det er kanskje ikke så problematisk 🤷🏼‍♂️